helm/templates/service.yaml

```diff
@@ -35,3 +33,39 @@ spec:
       name: grpc
   selector:
     {{- include "immudb.selectorLabels" . | nindent 4 }}
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: {{ include "immudb.fullname" . }}-replica-http
+  labels:
+    {{- include "immudb.labels" . | nindent 4 }}
+spec:
+  type: {{ .Values.service.type }}
+  ports:
+    - port: {{ .Values.replicaService.ports.http }}
+      targetPort: http
+      protocol: TCP
+      name: http
+    - port: {{ .Values.replicaService.ports.metrics }}
+      targetPort: metrics
+      protocol: TCP
+      name: metrics
+  selector:
+    {{- include "immudb.selectorLabels" . | nindent 4 }}
+---
```

Comment:
Using just one service for both master and replicas makes it impossible to split read and write requests. It would be best to define two different services: one for talking with the main instance, one for talking with replicas.